src/Rfc6455/Message.php

```diff
@@ -133,4 +135,21 @@ public function isOperation()
     {
         return in_array($this->getFirstFrame()->getOpcode(), [Frame::OP_TEXT, Frame::OP_BINARY]);
     }
+
+    /**
+     * @throws ControlFrameException
+     * @throws TooBigFrameException
+     */
+    public function checkControlFrameSize()
+    {
+        $frame = $this->getFirstFrame();
+
+        if (!$frame->isControlFrame()) {
```

Comment:
But IMO this should be throw by the Frame itself as the first frame knows if it's the last or not (thanks to the [first bit of the frame](https://tools.ietf.org/html/rfc6455#section-5.2))